ipaserver/install/server/replicainstall.py

```diff
@@ -1049,6 +1058,14 @@ def promote_check(installer):
                 logger.error("Certificates could not be provided when "
                              "CA is present on some master.")
                 raise ScriptError(rval=3)
+            if options.setup_ca and options.server and \
+               ca_host != options.server:
+                # Installer was provided with a specific master
+                # but this one doesn't provide CA
+                logger.error("The specified --server %s does not provide CA, "
+                             "please provide a server with the CA role",
+                             options.server)
+                raise ScriptError(rval=3)
```

Comment:
3 is documented to mean:

```
3 if the host exists in the IPA server or a replication agreement to the remote
master already exists
```
A new return value should probably be defined.

Reply:
Hi @rcritten 
thanks for the suggestion. Exit code 4 will be used for this specific error, man page also updated.